### PR TITLE
Nb add assessment preview

### PR DESCRIPTION
**Pull request by nick-benoit14**
Adds basic assessment preview

### CODE REVIEW

File: client/js/_author/components/assessments/navigation_bar_content.jsx [hunk 1]
```diff
@@ -2,48 +2,66 @@ import React            from 'react';
 import _                from 'lodash';
 import { hashHistory }  from 'react-router';
 import Icon             from '../bank_navigation/bank_icon';
+import appHistory       from '../../history';
 
-export default function AssessmentsView(props) {
+export default class NavigationBarContent extends React.Component {
```
Comment (jbasdf) [inline client/js/_author/components/assessments/navigation_bar_content.jsx]: Go back to function

File: client/js/_author/reducers/preview.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as Assessment from '../../actions/qbank/assessments';
```
Comment (jbasdf) [inline client/js/_author/reducers/preview.js]: This must die. Make it go away. Please daddy please.

File: client/js/middleware/qbank.js [hunk 3]
```diff
@@ -126,7 +126,14 @@ const qbank = {
     method : Network.GET,
     url    : url => `${url}`,
   },
-
+  [AssessmentConstants.GET_ASSESSMENT_PREVIEW]: {
+    method: Network.GET,
+    url: (url, action) => {
+      const bankId = encodeURIComponent(action.bankId);
+      const assessmentId = encodeURIComponent(action.assessmentId);
+      return `https://qbank-clix-dev.mit.edu/api/v1/assessment/banks/${bankId}/assessments/${assessmentId}/items?qti`;
```
Comment (jbasdf) [inline client/js/middleware/qbank.js]: don't hard code the url

File: client/js/middleware/serialization/qbank/serializers/multiple_choice.js [hunk 4]
```diff
@@ -74,8 +75,10 @@ export default function multipleChoiceSerializer(originalItem, newItemAttributes
   if (question) {
     newItem.question = {
       ...newItem.question,
-      ...serializeQuestion(originalItem.question, question)
+      ...serializeQuestion(originalItem.question, question),
+      genusTypeId: genusTypes.question[originalItem.type]
```
Comment (jbasdf) [inline client/js/middleware/serialization/qbank/serializers/multiple_choice.js]: Remove this. It must be done using the force. Only Old Spice Ben has the force.

File: client/js/_author/components/assessments/navigation_bar_content.jsx (hunk 1 above)
Comment (nick-benoit14) [inline client/js/_author/components/assessments/navigation_bar_content.jsx]: Looking at this again, I kind of think it has enough logic that it makes sense to be a class. but if you feel strongly about it I can change it

File: client/js/_author/reducers/preview.js (hunk 2 above)
Comment (nick-benoit14) [inline client/js/_author/reducers/preview.js]: Turns out this is important. We need to keep the data that comes back from qbank somewhere

**Pull request merged**